packages/cli/src/handlers/download.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-await-in-loop */
+import {
+    ApiChartAsCodeListResponse,
+    ApiChartAsCodeUpsertResponse,
+    AuthorizationError,
+    SavedChart,
+} from '@lightdash/common';
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import { getConfig } from '../config';
+import GlobalState from '../globalState';
+import { checkLightdashVersion, lightdashApi } from './dbt/apiClient';
+
+const DOWNLOAD_FOLDER = 'target';
```

Comment:
I picked this folder name because it is already included in our gitignore, but we can change it.

Reply:
`target` is a throwaway folder in a dbt project, so if this is run in a dbt directory the .yml will likely get deleted. 

What about `lightdash`?